hmf-common/src/test/resources/doid/example_doid.json

```diff
@@ -132,9 +132,6 @@
       "xrefs": [],
       "basicPropertyValues": []
     },
-    "equivalentNodesSets": [],
-    "logicalDefinitionAxioms": [],
-    "domainRangeAxioms": [],
-    "propertyChainAxioms": []
+    "logicalDefinitionAxioms": []
```

Comment:
I guess you may have had to add this one to make test succeed! But if this field is optional then code should also work without it.